### ISSUE TITLE
Icons not showing up for toasts

### ISSUE THREAD

**Issue by Elue-dev** [gunnartorfis/sonner-native#69]
**Describe the bug**
When I add a custom icon to the toast component, it did not show up, but not adding an icon at all makes it show

**To Reproduce**
```js
import { CircleCheck } from "lucide-react-native";


export function successToast({
  title,
  description,
  duration = 4000,
}: SuccessToast) {
  const successToastId = toast.success(title, {
    className: "bg-green-500",
    style: { backgroundColor: "green" },
    description,
    duration,
    icon: <CircleCheck />, // this does not show
  });
```

**Expected behavior**
The CircleCheck icon should show in the toast

| Library                      | Version |
| ---------------------------- | ------- |
| sonner-native                | 0.9.1  |
| react-native                 | 0.74.3   |
| react-native-reanimated      | 3.10.1  |
| react-native-gesture-handler | 2.16.1  |

**Comment by gunnartorfis** [comment]
Thanks for the report, that's quite urgent. 

I don't have much time to look at this during work hours today but I found out that the icon component passed is being logged as <CssInterpolate.XX />, I'm wondering if that has anything to do with it.

**Comment by gunnartorfis** [comment]
Fix has been released in 0.9.2
